sunpy/net/tests/test_fido.py

```diff
@@ -229,6 +230,16 @@ def test_vso_unifiedresponse():
     assert isinstance(uresp, UnifiedResponse)
 
 
+@pytest.mark.remote_data
+def test_vso_error():
+    with pytest.warns(SunpyUserWarning,
+        match="VSO-D404 Bad Request -- Invalid Time Sample : contains non digit"):
```

Comment:
This seems like an odd warning to be raised - the query below looks fine to me. Do you know why it raises this warning?

Reply:
I think it's https://github.com/sunpy/sunpy/issues/3331

Reply:
Yes it is #3331

Reply:
Do we want to have a test that might get invalidated down the road?

Reply:
Can you explain a little bit ? I didn't get it

Reply:
The test you wrote is using a bug to raise the warning which could get fixed in the near future.

Reply:
Oh yeah got it. I couldn't find another query so I used this .

Reply:
I got one that throws 404 Not Found error. Would that work ?

Reply:
Can you add it in addition to this test? Just add another with block.